tests/test_audit_outbox_postgres.sh

```
@@ -0,0 +1,351 @@
+WHERE outbox_delivery_record_id = '00000000-0000-4000-8000-000000000051'::uuid;
+SQL
+} 2>&1)"
+delivered_mutation_status=$?
+set -e
+if [[ ${delivered_mutation_status} -eq 0 || "${delivered_mutation_output}" != *"delivered outbox records are immutable"* ]]; then
+    echo "terminal delivery state accepted later mutation: ${delivered_mutation_output}" >&2
+    exit 1
+fi
+
+set +e
+atomicity_output="$({ psql "${DATABASE_URL}" -v ON_ERROR_STOP=1 -v canonical_event="${canonical_event}" <<'SQL'
```

Shellcheck (0.11.0)

[warning] atomicity_output appears unused. Verify use (or export if used externally).

(SC2034)